src/has.ts

```diff
@@ -0,0 +1,6 @@
+import global from 'dojo-core/global';
+import { add } from 'dojo-core/has';
+
+add('history', 'history' in global);
```

Comment:
Maybe there is a better name for this?  Again, maybe a styleguide thing @kfranqueiro in that we should try to name features in what sort of "standard" introduced them.  The History API was in theory HTML5, so would `html5-history` be a better name?

Also, while it maybe overkill, a more authoritative test would be `global.window && global.window.history && 'pushState' in global.window.history`.